.markdownlint-cli2.yaml

```diff
@@ -39,13 +39,13 @@ config:
 
   # Vertical whitespace helps the reader, so we should turn these on
   # again when someone has time to fix our existing files.
-  blanks-around-fences: true
-  blanks-around-headings: true
+  blanks-around-fences: false
+  blanks-around-headings: false
```

Comment:
From the comment it looks like openshift/enhancements wanted these on but couldn't for historical reasons. Any reason we shouldn't start with them? I can go either way...

Reply:
I'd keep this in sync 1:1 with openshift/enhancements. Basically I'd rather keep compatibility in case one of the EP in kni would have to move over there or the other way around. Besides that, checking the code this rule seems to have been there since start.